Modules/Course/classes/Objectives/Setup/ilCourseCleanupActivationMigration.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+/**
+ * This file is part of ILIAS, a powerful learning management system
+ * published by ILIAS open source e-Learning e.V.
+ *
+ * ILIAS is licensed with the GPL-3.0,
+ * see https://www.gnu.org/licenses/gpl-3.0.en.html
+ * You should have received a copy of said license along with the
+ * source code, too.
+ *
+ * If this is not the case or you just want to try ILIAS, you'll find
+ * us at:
+ * https://www.ilias.de
+ * https://github.com/ILIAS-eLearning
+ *
+ *********************************************************************/
+
+declare(strict_types=1);
+
+use ILIAS\Setup\Migration;
+use ILIAS\Setup\Environment;
+
+class ilCourseCleanupActivationMigration implements Migration
+{
+    private \ilDBInterface $db;
+
+    public function getLabel(): string
+    {
+        return "Remove Duplicate Roles In Activation Table";
```

Comment:
`Roles`, oder `Row` :-)?

```suggestion
        return "Remove Duplicate Rows In Activation Table";
```

Reply:
Thank you very much @mjansenDatabay ! Was probably thinking about the `AccessControl` part of this while writing.